edudoc/disciplinary-crash-course.md

```diff
@@ -0,0 +1,96 @@
+
+# Disciplinary Log and Banned Competitors
+
+* A list of banned competitors can be found in the [Disciplinary log}(https://www.worldcubeassociation.org/disciplinary). If a competitor attempts to attend a competition when they are banned from doing so, please contact the WDC.
+* Depending on the degree of severity and type of violation of the Code of Conduct, the WDC may ban a competitor from competing, or attending competitions. Such a ban will be communicated with local Delegates when it is issued. If a competitor is banned from competing, but not from attending competitions, they are permitted to attend competitions. They may also fulfill the roles of officials at competitions at the discretion of the WCA Delegate.
+* The WDC will post reports of particularly noteworthy cases that they have handled, as well as a quarterly digest to highlight smaller cases resolved in the last quarter. These are posted for the purpose of educating the community and WCA staff, as well as transparency. These will be posted in the announcement section on the WCA homepage and the WCA forum; a complete list of these posts and announcements can be found in the [Disciplinary log](https://www.worldcubeassociation.org/disciplinary).
+
+# Handling Behavioral Issues of Competitors at Competitions 
+
+* Delegates should primarily address minor behavioral issues of competitors at competitions themselves using [2j](https://www.worldcubeassociation.org/regulations/#2j), [2k](https://www.worldcubeassociation.org/regulations/#2k), and [2l](https://www.worldcubeassociation.org/regulations/#2l) of the regulations and applicable sub-points.
+  * If Delegates have not first tried to handle minor behavior issues by giving warnings and disqualifying competitors under regulation 2k, then the WDC is much less likely to apply any disciplinary action. We would like Delegates to attempt to handle minor incidents at a local level first before WDC gets involved. However, the WDC would like Delegates to report noteworthy behavioral issues so the WDC can record them. The WDC is always willing to give advice, and take disciplinary action as needed.
+* If any extraordinary cases occur, please describe them in the Delegate report.
+  * Please be aware the WDC will likely contact you for further information regarding the case.
+  * If an incident is discovered after the competition, this should be added to the Delegate report, or emailed to the Delegate report thread.
```

Comment:
> Delegate report thread

Capitalization here is a bit jarring. There's no style guide for how to refer to delegates, but I'd either capitalize both words like (Delegate Report) or none (delegate report).

Reply:
Other documents and the regulations use double capitals so I will adjust this to reflect that :)